tests/renderer/components/__snapshots__/commands-bisect-spec.tsx.snap

```diff
@@ -1,51 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Looks like we won't be needing snapshots anymore, right?

Reply:
We could always use inline Jest snapshots to explicitly test UI bits, but I don't see us using large snapshot files.

Reply:
Okey